dogs.html

```diff
@@ -0,0 +1,148 @@
+		</dl>
+		</div>
+
+		<div>
+		<img src="images/poppy-card.jpg" alt="a boy huges a dog image">
+		<dl>
+		<dt>Poppy </dt>
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+		<div>
+		<img src="images/jack-card.jpg" alt="a boy huges a dog image">
+		<dl>
+		<dt>Jack</dt>
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+		<div>
+		<img src="images/duffy-card.jpg" alt="a boy huges a dog image">
+		<dl>
+		<dt>Duffy </dt>
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+		<div>
+		<img src="images/lucas-card.jpg" alt="a boy huges a dog image">
+		<dl>	  
+		<dt>Lucas</dt>
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+		<div>
+		<img src="images/jake-card.jpg" alt="a boy huges a dog image">
+		<dl>
+		<dt>Jake </dt>
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+		<div>
+		<img src="images/angus-card.jpg" alt="a boy huges a dog image">
+		<dl>
+		<dt>Angus</dt>
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+		<div>
+		<img src="images/violet-card.jpg" alt="a boy huges a dog image">
+		<dl>
+		<dt>Violet </dt>
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+
+		<div>
+		<img src="images/piper-card.jpg" alt="a boy huges a dog image">
+		<dl>
+		<dt>Piper</dt>
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+
+		<div>
+		<img src="images/maximus-card.jpg" alt="a boy huges a dog image">
+		<dl>
+		<dt>Maximus</dt>
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+
+		<div>
+		<img src="images/luna-card.jpg" alt="a boy huges a dog image">
+		<dl>
+		<dt>Luna </dt>	
+		<dt>Cost to Adopt: $123.45</dt>
+		<dd>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</dd>
+		</dl>
+		</div>
+
+		<div>
+		<img src="images/stella-card.jpg" alt="a boy huges a dog image">
```

Comment:
Minor thing, but all of your alt tags are the same, but use different images. Are the images the same?